odxtools/nameditemlist.py

```diff
@@ -63,6 +63,9 @@ def append(self, item: T):
     def sort(self, key=None, reverse=False):
         return self._list.sort(key=key, reverse=reverse)
 
+    def __contains__(self, x) -> bool:
+        return x in self._list
```

Comment:
Looks a bit unrelated to PR

Reply:
true. drive-by fix. I think this one was caused by changing an ordinary list to a `NamedItemList` somewhere and the `in` operator was used on that list (a la `foo in bar`)